services/read.go

```diff
@@ -45,14 +51,10 @@ func GetStatesData() {
 
 }
 
-// Get cities function
+// ! Get cities function
 func GetCitiesData() {
-
-	conn := dbConn.DBConnection()
-	defer conn.Close()
-
 	// Get cities.

```

Comment:
also close the databse connection while function terminates

Reply:
Done